scripts/create-deploy-pr

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ORIGIN=origin
+DEPLOY_BRANCH=stages/prod
+
+if [ "${CI:-}" == "" ]; then
+    echo "This script is meant to be run in a continuous integration environment."
+    exit 1
+fi
+
+SHA=$(curl --silent https://idp.staging.login.gov/api/deploy.json | jq -r .git_sha)
+echo "Preparing deploy PR for ${SHA}"
+
+LAST_RC=$(gh release list --exclude-drafts --exclude-pre-releases --limit 1 | grep -E --only-matching 'RC [0-9]+(\.[0-9]+)?' | sed 's/RC //')
```

Comment:
I'm assuming that `gh` reads the `GH_TOKEN` env var? should we do a check before it?

```suggestion
if [[ -z "$GH_TOKEN" ]]; then
  echo "missing GH_TOKEN"
fi

LAST_RC=$(gh release list --exclude-drafts --exclude-pre-releases --limit 1 | grep -E --only-matching 'RC [0-9]+(\.[0-9]+)?' | sed 's/RC //')
```

Reply:
From my limited recollection of GitHub Actions, doesn't GitHub let you avoid having to create and manage your own token and provide one on your behalf? Can we use that instead and guarantee it exists?

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

Reply:
If you attempt to run `gh` in Github Actions it will [tell you](https://github.com/matthinz/identity-idp/actions/runs/7415590870/job/20179000919#step:5:9) that you need to set the GH_TOKEN environment variable. You set it to the token that Github has already created for you (why they don't do this automatically, I'm not sure).